misc/make-deps.sh

```diff
@@ -157,6 +157,9 @@ go get golang.org/x/lint/golint				# for `golint`-ing
 go get golang.org/x/tools/cmd/goimports		# for fmt
 go get github.com/kevinburke/go-bindata/go-bindata	# for compiling in non golang files
 go get github.com/dvyukov/go-fuzz/go-fuzz		# for fuzzing the mcl lang bits
+# heavily recommended to use curl to install instead of go get under https://golangci-lint.run/usage/install/
+curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.41.1
```

Comment:
why is this version hardcoded? can we use the latest version? is there a safer way to do this so we don't pipe to shell? if not, please change this so it only runs in ci.

Reply:
What is unsafe about piping into shell?

Reply:
I have changed this to something else. Any thoughts of it now?